packages/tags/src/Tags.ts

```diff
@@ -104,9 +110,27 @@ export class Tags extends FocusVisiblePolyfillMixin(SpectrumElement) {
         this.rovingTabindexController.clearElementCache();
     }
 
+    private async changeDefaultFocus(event: Event): Promise<void> {
+        await 0;
+        if (event.defaultPrevented) {
+            // No deleting when user prevents it.
+            return;
+        }
+        const items = [...this.tags] as Tag[];
+        /* c8 ignore next 3 */
+        if (items && !items.length) {
+            // no deleting when there aren't items.
+            return;
+        }
```

Comment:
This implies we're only reacting to the element removing itself, but there are other contexts that we should take into account here.
- when a user "prevents default" that will likely be in conjunction with the Tags component being controlled, so the element _may_ still be removed as part of the workflow
- the element becomes `disabled`
- etc.